app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -51,8 +51,18 @@ def submit_all_claim
       saved_claim = SavedClaim::DisabilityCompensation::Form526AllClaim.from_hash(form_content)
       saved_claim.save ? log_success(saved_claim) : log_failure(saved_claim)
       submission = create_submission(saved_claim)
-
-      jid = submission.start
+      # if jid = 0 then the submission was prevented from going any further in the process
+      jid = 0
+
+      # Feature flag to stop submission from being submitted to third-party service
+      # With this on, the submission will NOT be processed by EVSS or Lighthouse,
+      # nor will it go to VBMS,
+      # but the line of code before this one creates the submission in the vets-api database
+      if Flipper.enabled?(:disability_compensation_prevent_submission_job, @current_user)
+        Rails.logger.info("Submission ID: #{submission.id} prevented from sending to third party service.")
+      else
+        jid = submission.start
+      end
```

Comment:
This is great, thanks so much!

I mistakenly thought `form526_submission` model `start()` gets called much later in the process (after `direct_deposit`). As such, just wanted to confirm the `submit_all_claim` call happens before `direct_deposit`, right?

If not, awesome! If so, I'm wondering if we have a way to stop this after the PPIU and/or claims API calls

Also, can we include a serialized version of the submission object in the log?